subprojects/gst-plugins-bad/sys/d3d11/gstd3d11compositor.cpp

```diff
@@ -801,6 +822,41 @@ gst_d3d11_compositor_pad_set_property (GObject * object, guint prop_id,
           (GstD3D11CompositorSizingPolicy) g_value_get_enum (value);
       pad->position_updated = TRUE;
       break;
+    case PROP_PAD_CROP:
+    {
+        GST_DEBUG_OBJECT(pad, "setting crop parameters");
+       if (pad->crop_properties != NULL)
+        {
+            gst_structure_free(pad->crop_properties);
+            pad->crop_properties = NULL;
+        }            
+        pad->crop_properties =
+            gst_structure_copy(gst_value_get_structure(value));
+        if (pad->convert != NULL)
```

Comment:
if we don't enter this "if", we don't modify the properties
```
            pad->crop_left = left;
            pad->crop_top = top;
            pad->crop_right = right;
            pad->crop_bottom = bottom;
```
but set the
pad->apply_crop = TRUE;

later streaming thread will be reading this values unconditionally, so there will be an error if we don't enter this "if"